test/decl/ext/extensions.swift

```diff
@@ -366,3 +366,12 @@ protocol Rdar66943328 {
 }
 extension Rdar66943328 where Assoc == Int // expected-error {{expected '{' in extension}}
 #endif
+
+// Reject extension of existential type
```

Comment:
Could we add a test case using a type alias as well.
```swift
protocol P {}
typealias A = P

extension any A {}

// and also
typealias B = any P
extension B {}
```

This can make sure that diagnostic is mentioning user facing type `A (aka any 'P')`

Reply:
Thanks, good idea! Done.